salvo/src/lib/builder/test_nighthawk_builder.py

```diff
@@ -9,6 +9,8 @@
 from src.lib.builder import nighthawk_builder
 from src.lib import (cmd_exec, constants, source_tree, source_manager)
 
+_bazel_clean_cmd = "bazel clean"
```

Comment:
Since this is a constant, its name should be all-caps:

```python
_BAZEL_CLEAN_CMD = "bazel clean"
```

https://github.com/google/styleguide/blob/gh-pages/pyguide.md#s3.16-naming